lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -12,357 +12,365 @@
 require 'fluent/output'
```

Comment:
This should be `require "fluent/plugin/output"` according to http://docs.fluentd.org/v0.14/articles/plugin-update-from-v12#buffered-output-plugins